report-schema-v100.json

```diff
@@ -0,0 +1,600 @@
+                                                                    "type": "string",
+                                                                    "title": "The context schema",
+                                                                    "description": "The remediation itself.",
+                                                                    "default": "",
+                                                                    "examples": [
+                                                                        "Please update your scripts to be compatible with the changes."
+                                                                    ]
+                                                                }
+                                                            },
+                                                            {
+                                                                "type": {
+                                                                    "$id": "#/properties/detail/properties/remediations/items/anyOf/0/properties/type",
+                                                                    "type": "string",
+                                                                    "title": "The type schema",
+                                                                    "description": "Type of remediation.",
+                                                                    "default": "",
+                                                                    "enum": ["commands"],
+                                                                    "examples": [
+                                                                        "commands"
+                                                                    ]
+                                                                },
+                                                                "context": {
+                                                                    "$id": "#/properties/detail/properties/remediations/items/anyOf/0/properties/context",
+                                                                    "type": "array",
+                                                                    "title": "The context schema",
+                                                                    "description": "List of remediation commands.",
+                                                                    "default": [],
+                                                                    "examples": [
+                                                                        [
+                                                                            "rm -rf /tmp/42"
+                                                                        ]
+                                                                    ],
```

Comment:
This is wrong actually. E.g. 
```
{
  "type": "command",
  "context": [
    "leapp", "answer", "--section", "remove_pam_pkcs11_module_check.confirm=True"
  ]
}

```

We should add more examples to be able to validate the json schema better. Let's do it the next morning.